Python/Product/Analyzer/Intellisense/OutOfProcProjectAnalyzer.cs

```diff
@@ -1128,7 +1131,12 @@ class ImportStatementWalker : PythonWalker {
                 type = "class";
             }
             var startLoc = stmt.GetStart(ast);
-            var endLoc = stmt.GetStart(ast);
+            var endLoc = stmt.GetEnd(ast);

```

Comment:
FWIW, I'm pretty sure if this were the core of the problem then we'd have noticed it, as this code path gets hit _a lot_. But we should fix it.